lib/elixir/test/elixir/inspect_test.exs

```diff
@@ -154,6 +157,7 @@ defmodule Inspect.NumberTest do
 
   test "float" do
     assert inspect(1.0) == "1.0"
+    assert inspect(1234.001) == "1_234.001"
```

Comment:
Done in in https://github.com/elixir-lang/elixir/pull/4916/commits/4561587ac71802f40f72b4f730d1ea528bcbb13c